.changeset/spotty-dogs-hug.md

```diff
@@ -0,0 +1,52 @@
+---
+'example-app': minor
```

Comment:
This package is not published and can be skipped here.


Also no breaking changes are 'patch' now that we are at 0.x versions

Reply:
Ok. BTW, @freben  can you help me check the weird error?

```
packages/app/src/components/Root/Root.tsx:104:12 - error TS2322: Type '{ icon: OverridableComponent<SvgIconTypeMap<{}, "svg">>; to: string; text: string; }' is not assignable to type 'IntrinsicAttributes & SidebarItemProps'.
  Property 'children' is missing in type '{ icon: OverridableComponent<SvgIconTypeMap<{}, "svg">>; to: string; text: string; }' but required in type '{ to?: string | undefined; onClick?: ((ev: MouseEvent<Element, MouseEvent>) => void) | undefined; children: ReactNode; }'.

104           <SidebarItem
               ~~~~~~~~~~~
```

I couldn't find the changes to libraries other than the ones I changed. But it broke `yarn tsc`

Reply:
It sounds similar to https://stackoverflow.com/questions/61902377/react-typescript-error-for-material-ui-icons

Reply:
Essentially i think you'll find that there will be pretty significant work in making tsc happy. That's why we deferred this and asked for community help since we didn't have time to thoroughly work through it at this point.

Reply:
Let me figure out how to fix that.

Reply:
Indeed, it is such a pain to fix it. react-router made a lot of breaking changes before it went out of beta release.

Reply:
@freben I managed to fix the problem. However, I'm not sure about NavLink one. Can you help see whether I did it correctly?